README.md

```diff
@@ -99,60 +99,17 @@ interactive selection of the target object.
 
 [![asciicast](https://asciinema.org/a/e2gJS70bNEQrwMXEIA64SkpR1.svg)](https://asciinema.org/a/e2gJS70bNEQrwMXEIA64SkpR1)
 
-```sh
```

Comment:
Do you plan to move the ssh command samples to a different place? AFAIR it is the only explanation of pod selector options available for the users, so I would like to keep it in the documentation

Reply:
Sure! These are moved to the Helm chart's NOTES.

https://github.com/LeoColomb/IngreSsh/blob/222503413231b7c84b84e96aa2d7483956737f21/charts/ingressh/templates/NOTES.txt

These notes are printed at every install and are tweaked to show only the relevant information based on the config.
They can also be re-printed, as now documented on the README.

https://github.com/LeoColomb/IngreSsh/blob/222503413231b7c84b84e96aa2d7483956737f21/README.md#connecting